### PR TITLE
FCBHDBP-268 remove artisan commands related to V2 data sync, other unused cruft

### PR DESCRIPTION
**Pull request by vichugofsl**
# Description
It has removed artisan commands related to V2 data sync because they won't be used anymore.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-268

## How Do I QA This
- Run the [M] postman URL

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -16,19 +15,9 @@ class Kernel extends ConsoleKernel
...
-
-        Commands\BibleEquivalents\SyncBebliaBible::class,
-        Commands\BibleEquivalents\SyncDigitalBibleLibrary::class,
-        Commands\BibleEquivalents\SyncTalkingBibles::class,
-        Commands\BibleEquivalents\SyncEbible::class,
-        Commands\BibleEquivalents\SyncFcbhApk::class,
-        Commands\BibleEquivalents\SyncScriptureEarth::class,
-        Commands\BibleEquivalents\UpdateBibleLinkOrganizations::class,
-
         Commands\BibleFormats\FormatGetBible::class,
```
Comment (bradflood) [inline app/Console/Kernel.php:18]: please also remove FormatGetBible

File: app/Console/Kernel.php [hunk 2]
```diff
@@ -16,19 +15,9 @@ class Kernel extends ConsoleKernel
...
         Commands\BibleFormats\FormatRunberg::class,
```
Comment (bradflood) [inline app/Console/Kernel.php:19]: please also remove FormatRunberg

File: app/Console/Kernel.php [hunk 3]
```diff
@@ -39,30 +28,15 @@ class Kernel extends ConsoleKernel
 
         Commands\loaderPush::class,
```
Comment (bradflood) [inline app/Console/Kernel.php:29]: please remove loaderPush

File: app/Console/Kernel.php [hunk 4]
```diff
@@ -39,30 +28,15 @@ class Kernel extends ConsoleKernel
...
 
-        Commands\syncV2Database::class,
-        Commands\syncV2Users::class,
-        Commands\syncV2Profiles::class,
-        Commands\syncV2Bookmarks::class,
-        Commands\syncV2Highlights::class,
-        Commands\syncV2Notes::class,
-        Commands\reSyncV2Notes::class,
         Commands\syncV4Annotations::class,
```
Comment (bradflood) [inline app/Console/Kernel.php:31]: please remove syncV4Annotations

**Review by bradflood — CHANGES_REQUESTED** [review]
@vichugofsl I neglected to include some commands in the ticket. There are a few more that can be deleted

File: app/Console/Kernel.php (hunk 1 above)
Comment (vichugofsl) [reply app/Console/Kernel.php]: It has been removed.

File: app/Console/Kernel.php (hunk 2 above)
Comment (vichugofsl) [reply app/Console/Kernel.php]: It has been removed.

File: app/Console/Kernel.php (hunk 3 above)
Comment (vichugofsl) [reply app/Console/Kernel.php]: It has been removed.

File: app/Console/Kernel.php (hunk 4 above)
Comment (vichugofsl) [reply app/Console/Kernel.php]: @bradflood It has been removed.

**Review by bradflood — APPROVED** [review]

**Pull request merged**